Source/Shaders/BillboardCollectionFS.glsl

```diff
@@ -38,7 +73,32 @@ void main()
 #ifdef VECTOR_TILE
     color *= u_highlightColor;
 #endif
-
     gl_FragColor = color;
+
     czm_writeLogDepth();
+
+#ifdef CLAMP_TO_GROUND
+    if (v_eyeDepthAndDistance.x > -v_eyeDepthAndDistance.y) {
```

Comment:
How much of a pain would it be to add this to both the vertex and fragment shader?

We would want it to execute in the vertex shader when `ContextLimits.maximumVertexTextureImageUnits > 0`; otherwise, execute in the fragment shader. That condition just means we can read from textures in the vertex shader. The reason we would want to do this is for performance. Instead of discarding the fragment, you would put the vertex behind the eye so that it gets clipped like we do for distance display conditions or translucency by distance.

Reply:
I'm going to do this in a separate PR because the math is different since there's no `gl_FragCoord` in the vertex shader.  I'll open an issue